src/Graph.tsx

```diff
@@ -134,7 +135,9 @@ function getElkEdge(
     ],
     edge,
     sections: [],
-    layoutOptions: {},
+    layoutOptions: {
+      'elk.layered.priority.direction': isInitialEdge ? '1' : '0',
```

Comment:
could u add a comment here that would go into some details of this solution?

Reply:
Done